code/api/src/modules/user/resolvers.js

```diff
@@ -26,6 +27,16 @@ export async function create(parentValue, { name, email, password }) {
   }
 }
 
+// Update User
+export async function update(parentValue, { id, styleResult}) {
+  const user = await models.User.findOne({where: id })
+  return await user.update(
+    {
+      styleResult
+    }
+  )
+}
+
```

Comment:
This could be a thought for like an extension.
But should it authenticate the user first and only a logged in user can take the quiz or are we making it where anyone can take the quiz?